applications/nrf_desktop/doc/bas.rst

```diff
@@ -3,4 +3,27 @@
 GATT Battery Service
 ####################
 
-The ``bas`` module is used to control the GATT Battery Service.
+The ``bas`` module is used to define the GATT Battery Service and notify the subscribers about the battery level changes.
+
+Module Events
+*************
+
++---------------------+-------------------------+-------------+---------------+------------------+
+| Source Module       | Input Event             | This Module | Output Event  | Sink Module      |
++=====================+=========================+=============+===============+==================+
+| :ref:`battery_meas` | ``battery_level_event`` | ``bas``     |		      |			 |
```

Comment:
```suggestion
| :ref:`battery_meas` | ``battery_level_event`` | ``bas``     |		|		 |
```
Make the vertical lines aligned in the table, please.

Reply:
On my laptop the lines are aligned, I think its related to the way how Github displays it.

Reply:
these are probably tabs

Reply:
Should we use spaces instead of tabs in `*.rst` files ?